evento/src/components/TabContainer/index.js

```diff
@@ -6,18 +6,34 @@ const Tab = (props) => {
 	return (
 		<div className="Tab">
 			<h4>
-				<Link to={props.path}> {props.title} </Link>
+				<Link to={props.path} onClick={() => props.onClick(props.id)}> {props.title} </Link>
 			</h4>
 		</div>
 	);
 };
 
 class TabContainer extends Component {
+	constructor() {
+		super();
+		this.state = { selectedId: 1 };
+	}
+
+	onTabChange(tabId) {
+		this.setState({ selectedId: tabId });
+	}
+
 	render() {
 		return (
 			<div className="TabContainer">
 				{this.props.tabs.map(tab =>
-					<Tab key={tab.title} title={tab.title} path={tab.path} /> )}
+					<Tab
+						key={tab.title}
+						title={tab.title}
+						path={tab.path}
+						id={tab.id}
+						status={tab.id === this.state.selectedId ? 'selected' : 'not-selected'}
+						onClick={(e) => this.onTabChange(e)} 
```

Comment:
No need to add tab.id for the <Tab>. You can just in here call this.onTabChange(tab.id). Also that id seems pointless, why not just just use tab object as the selection definition. So just call: "this.onTabChange(tab)" and in status use "{tab == this.state.selectedTab ...}" etc.